cypress/e2e/Crosscutting/MegaMenu/MegaMenuDesktop.feature

```diff
@@ -0,0 +1,100 @@
+Feature: As a user, I want to be able to view and interact with the Mega Menu
```

Comment:
I don't see any tests that open the MM, make sure that analytics fire, nor make sure the items in the MM are as expected. Also you can do these tests agains A/B sections. 

Can you please update it to have complete tests or let me know if they live somewhere else.

Reply:
Analytics are in the same feature, at the end of a file

Added a test to verify items in MM ( using A)